src/clj/puppetlabs/services/jruby/jruby_puppet_core.clj

```diff
@@ -204,7 +207,12 @@
    agent-shutdown-fn :- IFn
    profiler :- (schema/maybe PuppetProfiler)
    metrics-service]
-  (let [initialize-pool-instance-fn (get-initialize-pool-instance-fn jruby-puppet-config profiler metrics-service)
+  (let [modified-jruby-puppet-config (if (get-in jruby-config [:multithreaded])
+                                       (do
+                                         (log/info "Disabling i18n for puppet because using multithreaded jruby")
```

Comment:
I know we're not really doing i18n anymore, but this should probably still be wrapped in `i18n/trs`. We were asked to keep the machinery working, and continuing to mark strings will save us work if we ever re-prioritize it.